minigalaxy/ui/gametile.py

```diff
@@ -252,7 +254,17 @@ def __download(self, download_info, finish_func, cancel_to_state):
             )
             self.download.append(download)
 
-        DownloadManager.download(self.download)
+        ds_ok = check_diskspace(total_file_size, Config.get("install_dir"))
```

Comment:
This doesn't really need a variable I think. It's only used once in the line below.